docs/tutorials/testing.md

```diff
@@ -38,13 +37,42 @@ For more extensive testing, you can leverage `prefect_test_harness` as a fixture
 import pytest
 from prefect.utilities.testing import prefect_test_harness
 
-from my_flows import my_favorite_function
+from my_flows import my_favorite_flow
 
 @pytest.fixture(autouse=True)
 def prefect_test_fixture():
     with prefect_test_harness():
         yield
 
-def test_my_favorite_function():
-    assert my_favorite_function().result() == 42
+def test_my_favorite_flow():
+    assert my_favorite_flow().result() == 42
+```
+
+## Unit testing tasks
+
+To test an individual task, you can create a flow inside of your test:
+
+```python
+from prefect import flow, task
+
+@task
+def my_favorite_task():
+    return 42
+
+def test_my_favorite_task():
+    @flow
+    def my_flow():
+        return my_favorite_task()
+    
+    state = my_flow().result()
```

Comment:
Nit: Naming this variable `task_state` disambiguates with state `my_flow().result()` is returning.
```suggestion
    task_state = my_flow().result()
```